src/optimizer/query_planner.c

```diff
@@ -3042,94 +2966,10 @@ qo_nljoin_cost (QO_PLAN * planp)
     {
       guessed_result_cardinality = (outer->info)->cardinality;
     }
-  inner_cpu_cost = guessed_result_cardinality * (double) QO_CPU_WEIGHT;
-  /* join cost */
-
-  if (qo_is_iscan (inner) && inner->plan_un.scan.index_equi == true)
-    {
-      /* correlated index equi-join */
-      inner_cpu_cost += inner->variable_cpu_cost;
-      if (qo_is_seq_scan (outer) && prm_get_integer_value (PRM_ID_MAX_OUTER_CARD_OF_IDXJOIN) != 0
```

Comment:
If the `PRM_ID_MAX_OUTER_CARD_OF_IDXJOIN` system parameter is no longer used, set the `PRM_DEPRECATED` flag.

Reply:
Okay. I fix it on next issue.